service/src/main/scala/ch/epfl/bluebrain/nexus/storage/client/StorageClient.scala

```diff
@@ -0,0 +1,264 @@
+package ch.epfl.bluebrain.nexus.storage.client
+
+import akka.NotUsed
+import akka.actor.ActorSystem
+import akka.http.scaladsl.client.RequestBuilding._
+import akka.http.scaladsl.model.Multipart.FormData
+import akka.http.scaladsl.model.Multipart.FormData.BodyPart
+import akka.http.scaladsl.model.Uri.Path.{Segment, Slash}
+import akka.http.scaladsl.model._
+import akka.http.scaladsl.model.headers.OAuth2BearerToken
+import akka.http.scaladsl.unmarshalling.{FromEntityUnmarshaller, Unmarshaller}
+import akka.stream.scaladsl.Source
+import akka.util.ByteString
+import cats.effect.{Effect, IO, LiftIO}
+import cats.implicits._
+import cats.effect.implicits._
+import ch.epfl.bluebrain.nexus.commons.http.HttpClient
+import ch.epfl.bluebrain.nexus.commons.http.HttpClient._
+import ch.epfl.bluebrain.nexus.commons.http.JsonLdCirceSupport._
+import ch.epfl.bluebrain.nexus.rdf.implicits._
+import ch.epfl.bluebrain.nexus.iam.client.IamClientError.{Forbidden, Unauthorized}
+import ch.epfl.bluebrain.nexus.iam.client.types._
```

Comment:
Do not use the iam client types, use the imports in
`import ch.epfl.bluebrain.nexus.iam.types._`
or 
`import ch.epfl.bluebrain.nexus.iam.acls.AccessControlLists`